Say Their Names/Source/Model/Person.swift

```diff
@@ -7,23 +7,30 @@
 //
 
 import Foundation
+import FirebaseFirestore
+import FirebaseFirestoreSwift
 
 struct Person: Codable {
-    var id: String
-    var fullName: String
-    var age: Int
-    var childrenCount: Int
-    var date: Date
-    var location: String
-    var media: [String]
-    var bio: String
-    var context: String
-    var donations: [Donation]
-    var petitions: [Petition]
+    let name: String
+    let age: Int
+    let numberOfChildren: Int
+    
+    // TODO: ensure what timezone dates are in
+    let dateOfIncident: Date
+    let dateOfBirth: Date
+
+    let location: DocumentReference
+    
+//    let media: [String]
+    let picture: String
+    let bio: String
+    let context: String
+    let donations: [DocumentReference]
+    let petitions: [DocumentReference]
     
     enum CodingKeys: String, CodingKey {
-        case id, fullName = "full_name", age, childrenCount = "children_count",
-        date, location, media, bio, context, donations, petitions
+        case /*id,*/ name , age, numberOfChildren = "number_of_children", dateOfBirth = "date_of_birth", 
```

Comment:
When working with Firebase in the past, I found it useful to keep a reference to the object ID. Should we keep ID as part of the objects?

Reply:
Each object (document) gets a unique documentID, so we don’t have to have a separate field. Though we can have a custom field too if we want to. Do you think it can be useful?

Reply:
I think it becomes easier to make related queries when the `id` is stored. For example, we could query against a different collection that has documents that keep a map of people and get back each of those documents that have the matching `personId`